ovirtlago/prefix.py

```diff
@@ -221,6 +221,8 @@ def flush(self):
 
             LOGGER.info('Results located at %s' % results_path)
 
+            if not result:
```

Comment:
I prefer to raise `TestFailedError` with some explanation about the failure rather than just exit.